tests/gbs/test_vibronic.py

```diff
@@ -67,6 +76,16 @@ def test_duschinsky(self):
         Ud = np.diag(self.wp ** -0.5) @ J @ np.diag(self.w ** 0.5)
         assert np.allclose(Ud, self.Ud)
 
+    def test_invalid_temperature(self):
+        """Test if function raises a ``ValueError`` when a negative temperature is given."""
+        with pytest.raises(ValueError, match="Temperature must be zero or positive"):
+            vibronic.gbs_params(self.w, self.wp, self.Ud, self.d, -1)
+
```

Comment:
Maybe a `test_zero_temperature` would be good, to check that the two mode squeezing parameters are zero in this case

Reply:
This is also why the coverage checks are failing

![image](https://user-images.githubusercontent.com/49409390/68301213-77d36780-006d-11ea-88f3-8f8e34672a9a.png)